pages/WriteReview/components/ReviewForm/__snapshots__/spec.jsx.snap

```diff
@@ -848,19 +848,19 @@ exports[`<ReviewForm /> should render form correctly 1`] = `
                     disabled={false}
                     flat={false}
                     onClick={null}
-                    testId="sendReviewButton"
+                    testId="Button"
```

Comment:
how are these changes related to your border radius changes? Please check your own pullrequest before submitting for not related files